src/overlays/actors/ovl_En_Bee/z_en_bee.c

```diff
@@ -15,12 +15,12 @@ void EnBee_Destroy(Actor* thisx, PlayState* play);
 void EnBee_Update(Actor* thisx, PlayState* play);
 void EnBee_Draw(Actor* thisx, PlayState* play);
 
-void func_80B5A9E8(EnBee* this, PlayState* play);
-void func_80B5AC3C(EnBee* this, PlayState* play);
-void func_80B5A854(EnBee* this);
-void func_80B5ABC4(EnBee* this);
+void EnBee_Idle(EnBee* this, PlayState* play);
+void EnBee_Hostile(EnBee* this, PlayState* play);
+void EnBee_SetupIdle(EnBee* this);
+void EnBee_SetupHostile(EnBee* this);
```

Comment:
These should be in the same order as they are in the file (somehow was missed in the original PR). The names of actionFuncs are usually based simple present tense verbs, though, would `FlyIdle` (Cf. EnFirefly) and `Attack` or `Dive` or something be appropriate?

Reply:
Yeah those work. it's a bit of a weird actor, the only difference between "idle" and "hostile" is that hostile bees are constantly updating their next fly-to-destination to be near Link, while idle bees set it once based on their home pos and never update it again.

Reply:
Is that the style we're going for in MM? Just want to note that it's the opposite recommendation in OoT:

https://github.com/zeldaret/oot/pull/1274#discussion_r895251012

Reply:
That's the first I've heard of it, that's what both have done in the past AFAIK.